### PR TITLE
fix: add jdk19 into night status check list

### PR DESCRIPTION
**Pull request by zdtsw**
nightlyBuild job does check all valid openjdk pipeline 
but only  report to slack on 8,11,17 - Unless this is on purpose
since we are releasing jdk19 good to have it in the list
do not need jdk-head/20 at the moment to get stable status.

### CODE REVIEW

**Comment by zdtsw** [comment]
@andrew-m-leonard not sure if it is intended to exclude 19 or we should add it to the list

**Comment by andrew-m-leonard** [comment]
@zdtsw adding 19 is good. Did you mean to change all the "whitespaces" ?

**Comment by zdtsw** [comment]
> @zdtsw adding 19 is good. Did you mean to change all the "whitespaces" ?

white space the is the auto fix from npm-groovy-lint.
kinda fix the lint indentation warnings

File: tools/nightly_build_and_test_stats.groovy [hunk 1]
```diff
@@ -19,147 +19,147 @@ import java.time.ZoneId
...
+    def apiUrl    = "${params.API_URL}"
+    def slackChannel = "${params.SLACK_CHANNEL}"
+    def featureReleases = [ 8, 11, 17, 19 ] // Consider making those parameters
+    def nightlyStaleDays = "${params.MAX_NIGHTLY_STALE_DAYS}"
+    def amberBuildAlertLevel = params.AMBER_BUILD_ALERT_LEVEL ? params.AMBER_BUILD_ALERT_LEVEL as Integer : -99
+    def amberTestAlertLevel  = params.AMBER_TEST_ALERT_LEVEL  ? params.AMBER_TEST_ALERT_LEVEL as Integer : -99
 
-  def healthStatus = [ "jdk8": null, "jdk11": null, "jdk17": null]
-  def testStats = []
+    def healthStatus = [ 'jdk8': null, 'jdk11': null, 'jdk17': null]
```
Comment (smlambert) [inline tools/nightly_build_and_test_stats.groovy:33]: You may also want to add jdk19 here and set to null

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**